agent/app/agent.go

```diff
@@ -490,8 +500,8 @@ func (agent *ecsAgent) registerContainerInstance(
 // reregisterContainerInstance registers a container instance that has already been
 // registered with ECS. This is for cases where the ECS Agent is being restored
 // from a check point.
-func (agent *ecsAgent) reregisterContainerInstance(client api.ECSClient, capabilities []*ecs.Attribute) error {
-	_, err := client.RegisterContainerInstance(agent.containerInstanceARN, capabilities)
+func (agent *ecsAgent) reregisterContainerInstance(client api.ECSClient, capabilities []*ecs.Attribute, clientToken string) error {
```

Comment:
Any reason using `clientToken` instead of `registrationToken` here? Seems that `registrationToken` is more meaningful to Agent 🤔

Reply:
I thought I did s/clientToken/registrationToken everywhere 😄
will change it